lib/Fmt_ast.ml

```diff
@@ -3744,25 +3790,27 @@ and fmt_constructor_declaration c ctx ~first ~last:_ cstr_decl =
 and fmt_core_type_gf c ctx typ =
   let {ptyp_attributes; _} = typ in
   let global_attr_opt, _ = split_global_flags_from_attrs ptyp_attributes in
-  ( match global_attr_opt with
```

Comment:
This is antiquated (from when we used `[@global]`), and causes issues with the new syntax.